Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1581,6 +1588,24 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
         FALSE,
         readWriteConstructParamFlags);
 
+    /**
+     * WebKitSettings:enable-websecurity:
+     *
+     * Enable or disable support for Web Security on pages.
+     *
+     * This setting is enabled by default.
```

Comment:
No need to mention this: the default value is part of the introspection metadata.